demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -543,9 +549,12 @@ def is_entity_version_match_playbook_version(self, implemented_entity_list_from_
                 # if entities with miss-matched versions were found and skipunavailable is
                 # not set or main playbook fromversion is below 6.0.0, fail the validation
                 if not is_version_valid and not skip_unavailable:
-                    invalid_version_entities.append(entity_name)
+                    entity_status[entity_id] = entity_status.get(entity_id, False)
+                else:
+                    entity_status[entity_id] = True  # valid or skipped
```

Comment:
since you use an `else` statement, let's de-morgan the condition for readability:
```python
if is_version_valid or skip_unavailable:
    entity_status[entity_id] = True
else:
    entity_status.setdefault(entity_id, False)
```

Reply:
Done